docs/user-guide/join-a-jitsi-meeting.md

```diff
@@ -4,11 +4,17 @@ title: Join a Jitsi Meeting
 sidebar_label: Join a Jitsi Meeting
 ---
 
+# Join by using a Jitsi link
 
-1. When visiting a Jitsi Meeting link, your browser may first ask you to grant microphone and/or camera access.
-   Please refert to the browser's documentation for details (e.g.
+People invite each other to Jitsi meetings mostly by sending a link.
+
+1. If you have received such an invite link from a trusted source,
```

Comment:
I would replace `trusted source` with `a person you trust`.

Reply:
:thinking: Hmm, now that I think about it, it might be good to mention both `a person you trust` and trust in the server/serverowner.

This way we could leave out `Trust in Server` below :point_down: .